openshift/patches/0006-patch-images.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/k8s/dialer.go b/k8s/dialer.go
+index 306e5b03..83027db1 100644
+--- a/k8s/dialer.go
++++ b/k8s/dialer.go
+@@ -24,7 +24,7 @@ import (
+ )
+ 
+ const (
+-	socatImage = "quay.io/boson/alpine-socat:1.7.4.3-r1-non-root"
++	socatImage = "registry.redhat.io/openshift-serverless-1/func"
```

Comment:
The name should be `registry.redhat.io/openshift-serverless-1/kn-plugin-func-rhel8`